rtloader/three/three.cpp

```diff
@@ -395,6 +395,41 @@ bool Three::getCheck(RtLoaderPyObject *py_class, const char *init_config_str, co
     return true;
 }
 
+bool Three::isCheckInitDeprecated(RtLoaderPyObject *py_class)
+{
+    PyObject *klass = reinterpret_cast<PyObject *>(py_class);
+    PyObject *init, *func_code, *co_varnames, *elt;
+    bool result = false;
+
+    // AgentCheck.__init__.__code__.co_varnames
+    //
+    init = PyObject_GetAttrString(klass, "__init__");
```

Comment:
We should clear python error if `PyObject_GetAttrString` returns NULL with `PyErr_Clear` (either in `done` or in each if).